### ISSUE TITLE
Training dataset

### ISSUE THREAD

**Issue by ZJUnewer** [ADLab-AutoDrive/BEVFusion#99]
Should the image and point cloud branches be trained with the training set only or with the training set and validation set together?
